src/main/menu.ts

```diff
@@ -45,7 +45,12 @@ function getHelpItems(): Array<MenuItemConstructorOptions> {
         const browserWindow = BrowserWindow.getFocusedWindow();
 
         if (browserWindow && !browserWindow.isDestroyed()) {
-          browserWindow.webContents.openDevTools({ mode: 'bottom' });
+          const { webContents } = browserWindow;
+          if (webContents.isDevToolsOpened()) {
+            browserWindow.webContents.closeDevTools();
+          } else {
+            browserWindow.webContents.openDevTools();
+          }
```

Comment:
If we removing the `options: { mode: 'bottom' }`, why not use the `toggleDevTools` ([docs](https://www.electronjs.org/docs/api/web-contents#contentstoggledevtools)) what do the same thing. If we still require the `options`, please return them back and that be look more reasonable.

Reply:
@HashimotoYT good catch, didn't see that API method. I don't know why the `mode: 'bottom'` was required there, maybe @felixrieseberg could chime in?